airbyte-integrations/connectors/destination-databricks/src/test-integration/kotlin/io/airbyte/integrations/destination/databricks/typededupe/AbstractDatabricksTypingDedupingTest.kt

```diff
@@ -26,25 +26,25 @@ import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Timeout
 import org.mockito.Mockito
 
-class DatabricksTypingDedupingTest : BaseTypingDedupingTest() {
+abstract class AbstractDatabricksTypingDedupingTest(
```

Comment:
adding separate test classes for the oauth and PAT configs, just to verify that they both work. This is super heavyweight :/ but I don't actually know which parts of our code were relying on the user/pass vs the PAT previously, so this lets us test (mostly) all the code paths

Reply:
Reasonable.  Files API and Jdbc both use PAT when switched to that auth mode, so staging and T+D code paths need to be tested.